data/_models/shared/Collections.csv

```diff
@@ -1,40 +1,41 @@
 tableName,tableExtends,columnName,columnType,key,required,refSchema,refTable,refLink,refBack,validation,semantics,description,profiles,visible
-Collections,,,,,,,,,,,dcat:Resource,"Generic listing of all resources. Should not be used directly, instead use specific types such as Databanks and Studies","DataCatalogueFlat,EMA",
+Collections,,,,,,,,,,,dcat:DatasetSeries,"Generic listing of all resources. Should not be used directly, instead use specific types such as Databanks and Studies","DataCatalogueFlat,EMA",
 Collections,,overview,heading,,,,,,,,,General information,DataCatalogueFlat,
-Collections,,id,,1,true,,,,,,dcterms:identifier,Internal identifier,"DataCatalogueFlat,EMA",
-Collections,,pid,,2,,,,,,,dcterms:identifier,Persistent identifier,"DataCatalogueFlat,EMA",
+Collections,,id,,1,true,,,,,,"dcterms:identifier,dct:identifier",Internal identifier,"DataCatalogueFlat,EMA",
+Collections,,pid,,2,,,,,,,"dcterms:identifier,dct:identifier",Persistent identifier,"DataCatalogueFlat,EMA",
 Collections,,acronym,,,,,,,,,dcterms:identifier,Acronym if applicable,"DataCatalogueFlat,EMA",
-Collections,,name,text,3,true,,,,,,"dcterms:title,foaf:name",Name used in European projects,"DataCatalogueFlat,EMA",
-Collections,,collection type,ontology,,,CatalogueOntologies,Collection types FLAT,,,,,,DataCatalogueFlat,
+Collections,,name,text,3,true,,,,,,"dcterms:title,foaf:name,dct:title",Name used in European projects,"DataCatalogueFlat,EMA",
+Collections,,collection type,ontology,,,CatalogueOntologies,Collection types FLAT,,,,"dcterms:type,dct:type",,DataCatalogueFlat,
 Collections,,local name,,,,,,,,,rdfs:label,"If different from above, name in the national language",DataCatalogueFlat,"['Cohort', 'Databank', 'Data source'].includes(collectionType?.name)"
 Collections,,network type,ontology_array,,,CatalogueOntologies,Network types,,,,dcterms:type,"Type of network, e.g. h2020 project",DataCatalogueFlat,['Network'].includes(collectionType?.name)
 Collections,,features,ontology_array,,,CatalogueOntologies,Network features,,,,,Characterizations of the network,"DataCatalogueFlat,EMA",['Network'].includes(collectionType?.name)
 Collections,,study type,ontology,,,CatalogueOntologies,Study types,,,,,Select 1 of the following types of study,"DataCatalogueFlat,EMA",['Study'].includes(collectionType?.name)
 Collections,,study type other,,,,,,,,,,"If other, describe the type of study","DataCatalogueFlat,EMA",['Study'].includes(collectionType?.name)
-Collections,,datasource type,ontology_array,,,CatalogueOntologies,Datasource types,,,,dcterms:type,Which of the following families of databanks best describe this data source,"DataCatalogueFlat,EMA","['Databank', 'Data source'].includes(collectionType?.name)"
```

Comment:
waarom is dit geen dcterms:type?

Reply:
use dct:type instead of dcterms:type?

Reply:
In health DCAT-AP a controlled vocabulary is proposed for dct:type: https://www.dublincore.org/specifications/dublin-core/dcmi-terms/#section-7

Reply:
Is `dct:type` then actually suitable for any of the columns? We want to specify the type of the collection, but with `dct:type` you can only specify that it _is_ a collection

Reply:
yes indeed, so this should just be standard for all entries in Collections and cannot be annotated in our data model.